src/simtools/applications/derive_limits.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/python3
+
+r"""
+Derives the limits for energy, radial distance, and viewcone using the LimitCalculator.
+
+The limits are derived based on the event loss fraction specified by the user.
+
+Command line arguments
+----------------------
+hdf5_file (str, required)
```

Comment:
Suggest to name the file not after the file format but after the content (e.g. `event_data_file`). You otherwise have to change many places in the code when deciding to use another file format.